### DISCUSSION TITLE
Is there any way to add indicator dots to the "Carousel with full width items"

### DISCUSSION THREAD

**Discussion by haydenkz** [saadeghi/daisyui#3062] [Q&A]
As the title states I'd like to add indicator dots to the carousel with full width items, I have tried to hack together the "Carousel with full width items" and "Carousel with indicator buttons" but I can't get it to work right. Any ideas would be greatly appreciated!!

**Comment by saadeghi** [comment]
Here's an example: https://codepen.io/saadeghi/pen/ExzWRdw?editors=1000
